src/countAnimals.js

```diff
@@ -1,7 +1,22 @@
-const data = require('../data/zoo_data');
+const { species } = require('../data/zoo_data');
+
+function defaultAnimals() {
```

Comment:
```suggestion
function getDefaultAnimals() {
```

É uma boa nomes de funções serem verbos, que indicam ação

Reply:
Vou me atentar sobre usar verbos nas funções! Obrigado!